src/iv/ivmain.cpp

```diff
@@ -115,8 +115,13 @@ main(int argc, char* argv[])
     
     bool use_ocio = color_space != "" && display != "" && view != "";
     std::string ocioenv = Sysutil::getenv("OCIO");
-    if (ocioenv.empty() || !Filesystem::exists(ocioenv))
-        setenv("OCIO", "ocio://default", 1);
+    if (ocioenv.empty() || !Filesystem::exists(ocioenv)) {
+#ifdef _MSC_VER
+    _putenv_s("OCIO", "ocio://default");
+#else
+    setenv("OCIO", "ocio://default", 1);
+#endif
```

Comment:
I suspect this needs some indentation to pass clang-format.

Reply:
hahaha, because we turn clang format off on line 39 and forget to turn it back on and the end of that function :-)

Reply:
added
// clang-format 
on on 78 line

and format a fix code